css/sass-lint/readme.md

```diff
@@ -0,0 +1,55 @@
+# Sass-Lint
+
+Sass-Lint is a pure node.js-only tool to help you write clean and consistent SCSS and Sass. You can run it manually from the command line, which will output any problems with your SCSS, including the filename and line number(s). Newly generated Adaptive.js projects use this tool by default, leveraging a Grunt task wrapper to invoke it.
```

Comment:
We should emphasize *Newly generated Adaptive.js projects use this tool by default, leveraging a Grunt task wrapper to invoke it.*, just in case–maybe make it the second sentence here, or put it in italics.